CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/BaseTest/Screens/ConfigurationScreen.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ConfigurationScreen.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import Foundation
+import XCTest
+
+public class ConfigurationScreen {
```

Comment:
[clarification]: this screen is not part of GiniCaptureSDK. Can we move it in the right place?

Reply:
We can do it in our meeting on Friday.